extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -98,12 +98,18 @@ export const buildOpenAttachmentElementLinkIcon = (element) =>
  */
 export const renderPageAttachmentUI = (pageEl, attachmentEl) => {
   if (isPageAttachmentUiV2ExperimentOn(pageEl.getAmpDoc().win)) {
-    if (attachmentEl.getAttribute('href')) {
+    // Outlinks can be an amp-story-page-outlink or the legacy version,
+    // an amp-story-page-attachment with an href.
+    const isOutlink =
+      attachmentEl.nodeName === 'AMP-STORY-PAGE-OUTLINK' ||
```

Comment:
not sure if there's any difference at all but we usually use `tagName`, both are probably fine, I have no idea to be honest.